tests/test_light_setup.py

```diff
@@ -11,7 +11,7 @@ def test_get_no_light_setup(sim):
 
 
 def test_set_default_light_setup(sim):
-    light_setup = [LightInfo(position=[1.0, 1.0, 1.0])]
+    light_setup = [LightInfo(position=[1.0, 1.0, 1.0, 0.0])]
```

Comment:
why the fourth parameter is not 1.0?

Reply:
Oh, I see.
"// use w == 1 for position or w == 0 for direction" 
Maybe we can put it here as a comment as well.